demos/browser/package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "amazon-chime-sdk-js": "file:../..",
-    "aws-sdk": "^2.771.0",
+    "aws-sdk": "^2.775.0",
```

Comment:
Nit: Seems like it was automatically/(unintentionally) added while testing with the demo app.